sign-in.txt

```diff
@@ -3,4 +3,9 @@
 //TODO: Please put your name and github username AS SHOWN below ... 
 //       ... then commit and create pull request to develop
 
-Brian Bansenauer : Bansenauer-Cascadia
\ No newline at end of file
+<<<<<<< issue12sruble
+Brian Bansenauer : Bansenauer-Cascadia
+Sim Ruble : sruble  
+=======
```

Comment:
This line is just showing you where your code ends and the develop branch code starts - remove it before you commit